dags/springer/parser.py

```diff
@@ -110,21 +112,23 @@ def __init__(self) -> None:
         super().__init__(extractors)
 
     def _get_abstract(self, article: ET.Element):
+        def is_latex_node(node: ET.Element):
+            return node.tag == "EquationSource" and node.attrib["Format"] == "TEX"
```

Comment:
would this take also the `<CDATA` text?

Reply:
As long as the CDATA is in the `<EquationSource format="TEXT>` tag yes. (It seems to be there)